src/Tabs.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab, onTabSelected }) => {
+  const selectedTabById = tabs.find(tab => tab.id === selectedTab) || null;
+
+  return (
+    <>
+      <ul className="options">
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            role="option"
+            tabIndex={0}
+            aria-selected={selectedTab === tab.id}
+            onClick={() => onTabSelected(tab)}
+            onKeyPress={(event) => (event.key === 'Enter') && onTabSelected(tab)}
```

Comment:
Make this a button, so you won't need all these properties

Reply:
It was interesting for me how to create custom element)